README.md

```diff
@@ -1,6 +1,8 @@
 # `isolate()`
 
-> A utility function to make scoped dataflow components in Cycle.js.
+> A utility function to make scoped dataflow components in Cycle.js. For example, this isolates DOM events seen by a component to those from it's own nodes, not the entire tree. 
```

Comment:
Thanks for the PR. :)
I'd like to keep this headline very short. Can you remove the second sentence?
Then I'd merge it.